### ISSUE TITLE
Create a training page

### ISSUE THREAD

**Issue by celestehorgan** [prometheus/docs#1599]
**What:**

Create a top-level page on prometheus.io to outline training options, similar to kubernetes.io/training. 

I propose a top level page with the following sections:

0) Stress upfront that Promethus is vendor neutral and does not endorse any company.

1) A section titled something along the lines of "Learn from the team/creators" linking @juliusv's [LF course](https://training.linuxfoundation.org/training/monitoring-systems-and-services-with-prometheus-lfs241/) and @brian-brazil's [training](https://training.robustperception.io/). 

2) A section titled "Other training resources" linking at a minimum [weaveworks' training page](https://www.weave.works/services/kubernetes-training/) and providing a button encouraging people to PR themselves onto the page. 

If the Prometheus team wants, we can also link any relevant results that Google turns up :).  

3) A section pointing people to the Prometheus Community page, where they can find consultants and other Prometheus related resources.

**Why:** 

This is something the CNCF (who I work for) is keen to see for prometheus.io. The community around Prometheus is robust and we've seen success creating a similar page for Kubernetes training programs, so we're eager to highlight.

As such, I'll be the one executing on this issue, but I want to make sure I have community consensus on the approach before I move ahead.

**Open questions to answer before proceeding:**

- [ ] Do we want to link training/certifications that are floating around the internet not 'officially' associated with the Prometheus project? Or do we want to limit this to training and services associated with companies/individuals listed on the Prometheus Community page?
- [ ] Are there any other training or certifications associated with Prometheus Community members that I haven't mentioned above?

**Comment by roidelapluie** [comment]
>     * [ ]  Do we want to link training/certifications that are floating around the internet not 'officially' associated with the Prometheus project? Or do we want to limit this to training and services associated with companies/individuals listed on the Prometheus Community page?

What we do for third party integrations usually is that we carefully review them and only list them when they respect our philosophy, guidelines & security rules.

I am not sure how that would work here, especially since the people that would more likely review that are also creators of such content, they would probably want to avoid looking at other parties IP in that matter? But if that is possible, I think we should take the time to review the trainings before listing them.

**Comment by celestehorgan** [comment]
@roidelapluie – I think that's a good point. 

The Kubernetes site handles it a bit differently because that community is much larger – the concept of a "training partner" exists and [has formal requirements](https://www.cncf.io/certification/training/becoming-a-ktp/). 

Julius (over email) seemed to think it would be fine if we just let people PR and do as they will. My opinion is there should be at least a _bit_ of a vetting process involved.

I'd suggest this: use a less rigorous set of requirements than KTP, and clearly documenting that on the page for potential PR-ers. Something like the following:

- Must have a publicly available syllabus or course outline
- Must provide x number of course hours/work
- If instructor led, must provide instructor credentials. 
- Any PRs against this page require 2x approvals by `prometheus-team` members, ideally not by people who have content/training already listed on the page.

... Or similar. I don't want to take the decision out of the contributor team's hands. 

Per your [governance](https://github.com/prometheus/docs/blob/master/content/governance.md#team-members) document, may I suggest bringing this to the `prometheus-team` mailing list and then letting me know how it goes?

**Comment by brian-brazil** [comment]
> What we do for third party integrations 

This varies based on what we're talking about. Our current standard for listing a company providing services is basically that they ask to be listed, and I don't see why we'd treat this differently - particularly given the difficulties in evaluating services in general.

From a website standpoint, does it really make sense to have a page whose sole purpose is to contain a list of 10-20 items? That seems a bit lacking in weight, and would be better as part of some other page.

**Comment by juliusv** [comment]
> From a website standpoint, does it really make sense to have a page whose sole purpose is to contain a list of 10-20 items? That seems a bit lacking in weight, and would be better as part of some other page.

Here's the Kubernetes example of it: https://kubernetes.io/training/. Some of the top info sections wouldn't apply to us of course. I don't know, I think it's ok though. It'll be a bit light for Prometheus initially, but making it a top-level link would really aid findability specifically for trainings. That said, I have a financial incentive for that page to exist (once I port my own materials from in-person to online), so it'd be good to hear other voices as well.

Overall I like the idea of letting anyone add themselves, but us checking that they follow the proposed requirements by @celestehorgan, to make sure that at least no complete bogus stuff gets added.

**Comment by brian-brazil** [comment]
I don't think we should be limiting things to online-only training. RP at least gets regular requests for in-person training, so it does seem like something that there's demand for.

I'd rather not get into the business of certifying training, and I'd hope that a providers landing page would provide sufficient information to entice a potential learner without us having to mandate things. I wouldn't want bureaucracy to exclude people like Julius or Tina if they wished to be listed.

We could have the commercial support list on this page too, there was some chatter about moving it iitc.

**Comment by juliusv** [comment]
I'm ok with having no checks at all as well (as was my initial stance) - just like for our existing "Commercial Support" section. And yeah, maybe we could combine both the new training and the existing Commercial Support section into one page to make it a bit more worthwhile. It'd still be nice to have a top-level menu item that makes both easily findable. "Support & Training"? That wouldn't look too bad at least:

![support_and_training](https://user-images.githubusercontent.com/538008/79337914-aca1ef80-7f26-11ea-9bab-b1c41c557312.png)

On online vs. in-person: agreed.

**Comment by roidelapluie** [comment]
![support_and_training](https://user-images.githubusercontent.com/538008/79337914-aca1ef80-7f26-11ea-9bab-b1c41c557312.png)

Then you need to include community support in that page ?

**Comment by brian-brazil** [comment]
We can link from one to the other.

**Comment by celestehorgan** [comment]
Thanks for the useful discussion, everyone! To summarize:

1) We like the idea of a Training page, but specifically if it's combined with Training & Support (and we list all resources on it). For now the shortlist will be those under [Commercial Support](https://prometheus.io/community/).

2) Very specifically, per @brian-brazil – we don't want to get into the business of certifying training providers. I think there should be clear messaging/wording on this page. Something like this?
   > These training programs are listed here for your convenience. All programs are created independently of the Prometheus project and their content is not governed or verified by the project's maintainers. 

3) We don't want to limit this to online training, so we need a section listing offline/consultant-led trainers. For now, I'll say the 'shortlist' of who we'll begin with is whomever is currently listed as a consultant [here](https://prometheus.io/community/). 

4) We want a section telling people how they can add their training, and we possibly? want some checks/balances in place in terms of what gets added. 

Action items from this:

1) @celestehorgan I'll go off and create a WIP/draft of this page with a placeholder section for how/who gets to add a training page.

2) I think you all need to debate amongst yourselves as to what (if any) governance should be around adding to this page, and then update your [governance doc](https://github.com/prometheus/docs/blob/master/content/governance.md) as required and let me know what you decide on.

**Comment by celestehorgan** [comment]
Also, @brian-brazil  – to address this comment because I think it's important:

> From a website standpoint, does it really make sense to have a page whose sole purpose is to contain a list of 10-20 items? That seems a bit lacking in weight, and would be better as part of some other page.

It may seem odd to do this (it is a fairly small amount of info!), but if anything it's a mark of Prometheus' maturity as a project and widespread use. Pages like this are as much about marketing as it is about findability. 

One of the CNCF's mission statements is to produce and support "enterprise grade" open source software. A huge part of selling any software as "enterprise" is about the resources available to the enterprise that adopts a software: will there be robust support if a critical bug is found? If I adopt this, how do I make my existing Ops engineers go from 0 to expert as quickly as possible? To compete against (true) enterprise-grade software that costs companies thousands of dollars a month in subscription fees, we need to act a bit like them. 

In short: it's a little silly, but it's a mark of maturity and widespread adoption that the Prometheus team should feel proud of :)

**Comment by brian-brazil** [comment]
> so we need a section listing offline/consultant-led trainers.

I think one training section is sufficient, rather than having things in the wrong section or both sections.

> I'll say the 'shortlist' of who we'll begin with is whomever is currently listed as a consultant here.

Let's not sign people up for things that they haven't asked for.

> We want a section telling people how they can add their training, and we possibly? 

I don't see a need to explain how to use github.

**Comment by juliusv** [comment]
>> I'll say the 'shortlist' of who we'll begin with is whomever is currently listed as a consultant here.
>>
> Let's not sign people up for things that they haven't asked for.

Yeah, the people and companies listed under "Commercial Support" don't necessarily offer trainings, so we'll need to keep those separate.

>> We want a section telling people how they can add their training
>
> I don't see a need to explain how to use github.

It totally makes sense to me to indicate that people can add themselves, and how. Of course that shouldn't include a whole GitHub tutorial, but something like "To add your own training offering here, please submit a GitHub pull request to <link to the file on GitHub>".

Agreed with the rest, don't mind too much about the page structuring.
